productcat/src/test/java/com/visenze/productcat/android/util/ResponseParserTest.java

```diff
@@ -17,6 +17,23 @@
 
 public class ResponseParserTest {
 
+
+    @Test
+    public void parseSRPResult() {
+        String responseString = "{\"im_id\":\"202002063654d637a048cbb5106e7c28d05dacf41d9720acf38.jpg\",\"reqid\":\"05O1IQO3OSUHB8VCQ0Q536PG\",\"status\":\"OK\",\"error\":[],\"product_types\":[{\"type\":\"bag\",\"score\":0.862,\"box\":[32,2,185,218],\"attributes\":{}}],\"srp_url\":\"https://shopping.visenze.com/search/05O1IQO3OSUHB8VCQ0Q536PG?country=SG&amp;uid=4444\",\"recognize_result\":[{\"tag_group\":\"category\",\"tags\":[{\"tag_id\":\"root|bag\",\"tag\":\"bag\",\"score\":0.9836792349815369},{\"tag_id\":\"root|bag|backpack\",\"tag\":\"backpack\",\"score\":0.9464564323425293}]}]}";
+        ResultList resultList = ResponseParser.parseResult(responseString);
+
+        assertEquals("https://shopping.visenze.com/search/05O1IQO3OSUHB8VCQ0Q536PG?country=SG&amp;uid=4444", resultList.getSrpUrl());
```

Comment:
can change this to "https://shopping.visenze.com/search/05O1IQO3OSUHB8VCQ0Q536PG?country=SG&uid=4444" and also the responseString.

The Wiki page auto convert "&" to "&amp;".

Reply:
changed already